data-raw/chicago_taxi.R

```diff
@@ -0,0 +1,43 @@
+## code to prepare `chicago_taxi` dataset goes here
+
+library(tidyverse)
+library(tidymodels)
+library(janitor)
+
+# https://data.cityofchicago.org/Transportation/Taxi-Trips-2022/npd7-ywjz
+taxi_raw <- read_csv("~/Downloads/Taxi_Trips_-_2022.csv") |>
```

Comment:
I think that you can get a url for the api. That's what we did with the train data. [This url](https://data.cityofchicago.org/api/views/e55j-2ewb/rows.csv?accessType=DOWNLOAD) should work (and you can probably get one with parameters too).

Reply:
[`0275faa` (#10)](https://github.com/tidymodels/modeldatatoo/pull/10/commits/0275faa5a9aa948c40747c023e9aa8390e5feb1b)